src/Symfony/Component/Form/Extension/Validator/Constraints/Form.php

```diff
@@ -26,6 +28,12 @@ class Form extends Constraint
         self::NO_SUCH_FIELD_ERROR => 'NO_SUCH_FIELD_ERROR',
     ];
 
+    #[HasNamedArguments()]
```

Comment:
I'm not sure about our CS, but parenthesis are optional.

```suggestion
    #[HasNamedArguments]
```